test/modules/linear_boltzmann_solvers/transport_keigen/keigenvalue_transport_3d_openmcxs_U235_smm.lua

```diff
@@ -0,0 +1,105 @@
+-- 3D 17G KEigenvalue::Solver test using power iteration and OpenMC MGXS library
```

Comment:
172G

Reply:
Maybe 76G, since that's the actual number of groups used by the test?

Reply:
Fixed.